### ISSUE TITLE
Nx Storybook plugin not source mapping correctly

### ISSUE THREAD

**Issue by parksj10** [nrwl/nx#4979]
## Current Behavior

Storybook is not source mapping, presents as bundled js
## Expected Behavior

Storybook should show the same, source mapped, traceable errors when in development mode

## Steps to Reproduce

Init as per documentation

Implement [workaround](https://github.com/nrwl/nx/issues/4942#issuecomment-794406363) to enforce development mode ([other issue](https://github.com/nrwl/nx/issues/4942))

### Failure Logs

example error in storybook:
```
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: null.
    at createFiberFromTypeAndProps (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:204846:21)
    at createFiberFromElement (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:204874:15)
    at reconcileSingleElement (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:193845:23)
    at reconcileChildFibers (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:193905:35)
    at reconcileChildren (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:196783:28)
    at mountIndeterminateComponent (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:197683:5)
    at beginWork (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:198842:16)
    at HTMLUnknownElement.callCallback (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:183738:14)
    at Object.invokeGuardedCallbackDev (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:183787:16)
    at invokeGuardedCallback (http://localhost:4400/vendors~main.f222bc07ab398a42ec88.bundle.js:183849:31)
```

### Environment

```
Node : 15.6.0
  OS   : darwin x64
  npm  : 6.14.11
  
  nx : Not Found
  @nrwl/angular : Not Found
  @nrwl/cli : 11.3.0
  @nrwl/cypress : 11.3.0
  @nrwl/devkit : 11.3.0
  @nrwl/eslint-plugin-nx : 11.3.0
  @nrwl/express : Not Found
  @nrwl/jest : 11.3.0
  @nrwl/linter : 11.3.0
  @nrwl/nest : 11.3.0
  @nrwl/next : Not Found
  @nrwl/node : 11.3.0
  @nrwl/react : 11.3.0
  @nrwl/schematics : Not Found
  @nrwl/tao : 11.3.0
  @nrwl/web : 11.3.0
  @nrwl/workspace : 11.3.0
  typescript : 4.1.5
```

**Comment by parksj10** [comment]
still an issue...

**Comment by mandarini** [comment]
Hi @parksj10 , do you think that this is a duplicate of [#4942](https://github.com/nrwl/nx/issues/4942) or [#5070](https://github.com/nrwl/nx/issues/5070)?

**Comment by parksj10** [comment]
not precisely, as stated 

[> The above error occurred after I implemented your workaround (I.e. forcing to development), which I thought would start yielding the source mapped error output (like I see when I actually run the app in development mode, instead of storybook)](https://github.com/nrwl/nx/issues/4942#issuecomment-794475986)

this is only getting It to _run_ in development mode, not emit correct source mappings. I _believe_ this a separate issue, that would likely have to be addressed after [#4942](https://github.com/nrwl/nx/issues/4942)

**Comment by juristr** [comment]
Hey @parksj10, we just merged #5313 last week which should fix this. Just tested locally and seems to be fine.

![image](https://user-images.githubusercontent.com/542458/115844219-e94cd480-a41f-11eb-8e8c-edb5ca4223d2.png)

In case this is still an issue after it gets released, feel free to reopen. And thx for reporting this 🙂
